### DISCUSSION TITLE
Specific reason behind Maui not support Web & Linux ?

### DISCUSSION THREAD

**Discussion by manishbisen** [dotnet/maui#8750] [General]
When Uno & Avalonia products which are based on C# can support Web & Linux platforms ,then why Microsoft own product which is Maui & which is also based on C# & Xmal cant support to Web & Linux.Is there any specific reason behind this ?

**Comment by Xyncgas** [comment]
Maui supports web via blazor and linux via xamarin

**Reply by manishbisen** [reply]
I mean Maui with Xmal target to web.
And I think your point related to Linux via xamarin make nosense.
